bitrise.yml

```diff
@@ -13,14 +13,17 @@ workflows:
     title: Runs tests
     steps:
     - go-list: { }
-    - golint: { }
-    - errcheck: { }
-    - go-test: { }
-    - codecov:
-        run_if: .IsCI
+    - script:
+        title: golangci-lint
```

Comment:
Why don't we use the internal check step? I think it comes with our standard lint config, similarly to the Bitrise CLI: https://github.com/bitrise-io/bitrise/blob/master/bitrise.yml#L12-L15 ?

Reply:
I don't think it's a good idea for binary projects: it hardcodes a golangci-lint version, which is good for step repos, but bad for binary repos like this where we use a more recent Go version. I had issues with this in the past, upgrading the Go version required a newer linter version, which is not possible when using the check step.